spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/util/ExceptionUtils.java

```diff
@@ -19,13 +19,28 @@
 import org.apache.commons.logging.Log;
 
 /**
+ * <p>
+ *     Utility class for logging exceptions. Useful for test purposes -
+ *     when a warning message should be presented an exception can be thrown.
```

Comment:
Maybe.. this allows tests to be more strict, for example throwing an exception as opposed to just logging a warning.

ps is this the same idea as the assert keyword?